docker-builds/server/config.example.edn

```diff
@@ -0,0 +1,54 @@
+{:emailer {:private-key "25677d268904ea651f84e37cfd580696c5c793dcd9730c415bf03b96003c09e9ef8"
```

Comment:
What about this emailer private key? Do I need to setup anything for this to work?

Reply:
I don't know about it, I think we're not too interested in this functionality right now, and district0x should set it up to whatever they want on their deploy.